client_plugin/utils/plugin_server/plugin_server_linux.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 const (
-	mountRoot     = "/mnt/vmdk"           // VMDK and photon volumes are mounted here
-	pluginSockDir = "/run/docker/plugins" // Unix sock for the plugin is maintained here
+	// Unix sock for the plugin is maintained here
+	pluginSockDir = "/run/docker/plugins"
```

Comment:
nit: since we only have a single `const`, we could have it without the `()` block.